### DISCUSSION TITLE
How to operate with the response

### DISCUSSION THREAD

**Discussion by alexei-moneyr** [postmanlabs/postman-flows#263] [Q&A]
Hey, please help to understand how to work with response I'm getting?
for example saving in variable (data block) any data from response. How can I parse it to different variables?! 

/data/body/status giving me red mark with error "should resolve to string instead of "unknown"

**Comment by samay-v** [comment]
Hi @alexei-moneyr
Some questions to help me help you with this:
1. Which block are you using to save in a variable? 
2. Does your request have an example saved in it?

**Reply by itseramin** [reply]
Same issue... can't use the 'find' operator if all the data in the response is 'unknown'... Any tips?

**Reply by alexei-moneyr** [reply]
simple, sending data to my api, getting the response and trying to use the result.

![image](https://user-images.githubusercontent.com/82156498/156778789-be2352c2-7e14-4512-a2c1-2e03f294dabe.png)
